distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/ImagingTest.java

```diff
@@ -0,0 +1,196 @@
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.keepRuntimeFolder;
+
+import static com.jayway.restassured.RestAssured.delete;
+import static com.jayway.restassured.RestAssured.given;
+import static com.jayway.restassured.RestAssured.when;
+
+import static ddf.test.itests.AbstractIntegrationTest.DynamicUrl.SECURE_ROOT;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Dictionary;
+import java.util.Hashtable;
+
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.http.HttpStatus;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.ops4j.pax.exam.Option;
+import org.ops4j.pax.exam.junit.PaxExam;
+import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
+import org.ops4j.pax.exam.spi.reactors.PerClass;
+import org.osgi.service.cm.Configuration;
+
+import ddf.common.test.BeforeExam;
+import ddf.test.itests.AbstractIntegrationTest;
+
+@RunWith(PaxExam.class)
+@ExamReactorStrategy(PerClass.class)
+public class ImagingTest extends AbstractIntegrationTest {
+    private static final String[] REQUIRED_APPS =
+            {"catalog-app", "solr-app", "spatial-app", "imaging-app"};
+
+    private static final String TEST_IMAGE_NITF = "i_3001a.ntf";
+
+    private static final String TEST_MTI_NITF = "gmti-test.ntf";
+
+    @Override
+    protected Option[] configureDistribution() {
+        return options(karafDistributionConfiguration(maven().groupId(
+                "org.codice.alliance.distribution")
+                .artifactId("alliance")
+                .type("zip")
+                .versionAsInProject()
+                .getURL(), "alliance", KARAF_VERSION).unpackDirectory(new File("target/exam"))
+                .useDeployFolder(false));
+    }
+
+    @Override
+    protected Option[] configureCustom() {
+        return options(wrappedBundle(mavenBundle("ddf.test.itests", "test-itests-ddf").classifier(
+                "tests")
+                .versionAsInProject()).bundleSymbolicName("test-itests-ddf")
+                .exports("ddf.test.itests.*"), keepRuntimeFolder());
+    }
+
+    @BeforeExam
+    public void beforeAllianceTest() throws Exception {
+        try {
+            basePort = getBasePort();
+            setLogLevels();
+            getServiceManager().waitForRequiredApps(REQUIRED_APPS);
+            getServiceManager().waitForAllBundles();
+            getCatalogBundle().waitForCatalogProvider();
+            configureSecurityStsClient();
+        } catch (Exception e) {
+            LOGGER.error("Failed in @BeforeExam: ", e);
+            fail("Failed in @BeforeExam: " + e.getMessage());
+        }
+    }
+
+    private String ingestNitfFile(String fileName) throws Exception {
+        InputStream inputStream = getClass().getResourceAsStream("/" + fileName);
```

Comment:
I think this should be `getClass().getClassLoader().getResourceAsStream(...)` for Windows Compatibility. I think the `/` is discourage as well. https://codice.atlassian.net/wiki/display/DDF/Windows+Compatibility .
@Lambeaux